components/PostAuth/index.jsx

```diff
@@ -26,10 +27,18 @@ const Form = styled.form`
   flex-grow: 1;
 `
 
+const Highlight = styled.span`
+  font-size: ${(props) => props.theme.fontSizes[3]};
+  border-radius: ${(props) => props.theme.radii[6]};
+  padding: 0rem 1rem;
+  margin-right: 0.5rem;
+  background-color: #ffc0cb;
+`
```

Comment:
Should we make this into a shared component? And pass `font-size` as a prop that just gets forwarded? Any reason not to?

Reply:
Sure, why not.